src/Proto3/Suite/DotProto/Generate.hs

```diff
@@ -87,7 +85,7 @@ import qualified GHC.Types.Basic                as GHC (PromotionFlag(..))
 import           GHC.Hs                         (HsSigType(..))
 import           GHC.Parser.Annotation          (noLocA)
 #if !MIN_VERSION_ghc(9,6,0)
-import qualified GHC.Types.SourceText           as GHC
+import qualified GHC.Types.SourceText           as GHC ()
```

Comment:
Do we still need type class instance or something?

Reply:
No, we don't. I removed this import.